src/main/golem/util/validation/README.md

```diff
@@ -0,0 +1,102 @@
+# Matrix Validation DSL
+
+This package provides definitions for a domain-specific language that you can use to validate matrices used as inputs to your functions in a consistent way. The rules throw detailed, human-readable exceptions using an appropriate Java exception class based on rules that look like this:
+
+```kotlin
```

Comment:
I have no idea what this means.


> On 29 Mar 2016, at 11:20 PM, Kyle Kauffman <notifications@github.com> wrote:
> 
> In src/main/golem/util/validation/README.md:
> 
> > @@ -0,0 +1,102 @@
> > +# Matrix Validation DSL
> > +
> > +This package provides definitions for a domain-specific language that you can use to validate matrices used as inputs to your functions in a consistent way. The rules throw detailed, human-readable exceptions using an appropriate Java exception class based on rules that look like this:
> > +
> > +```kotlin
> 
> Adding relevant includes to the README would be helpful.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>